playbooks/openstack/README.md

```diff
@@ -143,7 +150,18 @@ $ vi inventory/group_vars/all.yml
 4. Set the `openshift_openstack_default_flavor` to the flavor you want your
    OpenShift VMs to use.
    - See `openstack flavor list` for the list of available flavors.
+5. If you opt to use Kuryr for the networking, make sure that you review all
+   the kuryr options in the file. At the very least, if you use Kuryr, you
+   should uncomment:
 
+```bash
+#openshift_use_kuryr: True
+#openshift_use_openshift_sdn: False
+#os_sdn_network_plugin_name: cni
+#openshift_node_proxy_mode: userspace
+#openshift_hosted_manage_registry: false
+#openshift_hosted_manage_router: false
```

Comment:
We should add here also:
#use_trunk_ports: true

Reply:
We should also mention that this requires the Trunk ports extension (which I didn't have enabled). This guide describes how to detect and validate it's enable:

https://docs.openstack.org/neutron/pike/admin/config-trunking.html

I wonder if we can check for this in our prerequisites (as a separate PR).

Reply:
Done. I'll update the patch